src/uwtools/tests/config/test_core.py

```diff
@@ -867,37 +824,30 @@ def test_Config_characterize_values(nml_cfgobj):
     assert template == ["    p3: {{ n }}"]
 
 
-def test_Config_reify_scalar_str(nml_cfgobj):
-    for x in ["true", "yes", "TRUE"]:
-        assert nml_cfgobj.reify_scalar_str(x) is True
-    for x in ["false", "no", "FALSE"]:
-        assert nml_cfgobj.reify_scalar_str(x) is False
-    assert nml_cfgobj.reify_scalar_str("88") == 88
-    assert nml_cfgobj.reify_scalar_str("'88'") == "88"  # quoted int not converted
-    assert nml_cfgobj.reify_scalar_str("3.14") == 3.14
-    assert nml_cfgobj.reify_scalar_str("NA") == "NA"  # no conversion
-    assert nml_cfgobj.reify_scalar_str("@[foo]") == "@[foo]"  # no conversion for YAML exceptions
-    with raises(AttributeError) as e:
-        nml_cfgobj.reify_scalar_str([1, 2, 3])
-    assert "'list' object has no attribute 'read'" in str(e.value)  # Exception on unintended list
-
-
-def test_Config_dereference_unexpected_error(nml_cfgobj):
-    exctype = FloatingPointError
-    with patch.object(core.J2Template, "render", side_effect=exctype):
-        with raises(exctype):
-            nml_cfgobj.dereference(ref_dict={"n": "{{ n }}"})
+# def test_Config_reify_scalar_str(nml_cfgobj):
+#     for x in ["true", "yes", "TRUE"]:
+#         assert nml_cfgobj.reify_scalar_str(x) is True
+#     for x in ["false", "no", "FALSE"]:
+#         assert nml_cfgobj.reify_scalar_str(x) is False
+#     assert nml_cfgobj.reify_scalar_str("88") == 88
+#     assert nml_cfgobj.reify_scalar_str("'88'") == "88"  # quoted int not converted
+#     assert nml_cfgobj.reify_scalar_str("3.14") == 3.14
+#     assert nml_cfgobj.reify_scalar_str("NA") == "NA"  # no conversion
+#     assert nml_cfgobj.reify_scalar_str("@[foo]") == "@[foo]"  # no conversion for YAML exceptions
+#     with raises(AttributeError) as e:
+#         nml_cfgobj.reify_scalar_str([1, 2, 3])
+#     assert "'list' object has no attribute 'read'" in str(e.value)  # Exception on unintended list
```

Comment:
The method-under-test became a function in `uwtools.config.jinja2` -- I'll reactivate this test when breaking this module up to mirror the actual library modules.

Reply:
Test reactivated.